src/content/posts/2025-call-for-volunteers.mdx

```diff
@@ -0,0 +1,81 @@
+- ⚡ **Infrastructure Team**: Ensure the technical side of the conference runs smoothly, including managing online platforms and tools, including GitHub, Discord, and the streaming.
+- 💻 **Website Team**: Help maintain and update the conference website, ensuring it is user-friendly and informative.
+- 🤝 **Volunteer Coordination Team**: Assist in recruiting, and coordinating volunteers to ensure a smooth and successful event.
+
+Each role offers unique learning opportunities and direct contribution to our mission. For detailed responsibilities and requirements, visit our [documentation](https://conference.pyladies.com/docs/roles_and_responsibilities/).
+
+## Meet your Team Leads
+Our experienced team leads will guide and support you throughout your volunteer journey:
+
+### Denny Perez - Sponsorship & Communications Lead
+<img src="/images/authors/denny.jpg" alt="Denny Perez" style={{maxWidth: "400px", height: "auto"}} />
+- **Bio**: Software QA Analyst. Python community manager. Python Software Foundation (PSF) Fellow & Director. Advocates diversity and inclusion within tech communities. PyLadies chapters organizer.
+- **What Denny loves about PyLadiesCon**: #todo: Personal quote about the conference impact
+
+### Georgi Ker - Design & Media Lead
+<img src="/images/authors/georgi.jpg" alt="Georgi Ker" style={{maxWidth: "400px", height: "auto"}} />
+- **Bio**: Mother of 2 adults | Director @PSF | Fellow Member | 2020 PSF community service award | PSF D&I Workgroup Chair | PyCon US Design Lead since 2022.
+- **What Georgi loves about PyLadiesCon**: # todo: Personal quote about the conference impact
+
+### Abigail Dogbe - Programme Lead
+<img src="/images/authors/abigail.jpg" alt="Abigail Dogbe" style={{maxWidth: "400px", height: "auto"}} />
+- **Bio**: Contributor and community member in the African Python Software Community, Abigail has served as an organizer for local and international community events. She is a Fellow and a Community Service award winner at the PSF.- **Bio**: Contributor and community member in the African Python Software Community, Abigail has served as an organizer for local and international community events. She is a Fellow and a Community Service award winner at the PSF.
+- **What Abigail loves about PyLadiesCon**: PyLadiesCon is a great opportunity to learn about the work that PyLadies are doing all over the world.
+
+### Cristián - Infrastructure Lead
+<img src="/images/authors/cristian.jpg" alt="Cristián" style={{maxWidth: "400px", height: "auto"}} />
+- **Bio**: Python community and conference serial organizer, contributing to many Open Source initiatives. He is a PSF Board Director and Fellow. He is currently working at The Qt Company.
+- **What Cristián loves about PyLadiesCon**: #todo: Personal quote about the conference impact
```

Comment:
@cmaureir  Please add your personal quote here on what you love about PyLadiesCon.